packages/components/inputs/search-text-input/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 commercetools GmbH
```

Comment:
You can remove the year

Reply:
Updated [975e64c](https://github.com/commercetools/ui-kit/pull/2411/commits/975e64c4e6fe3f4fc12ded6f2dcf9c22b6ee339b)